eips_code/eip4750_test.py

```diff
@@ -0,0 +1,82 @@
+    assert validate_function(0, bytes((OP_NUMBER, OP_POP, OP_NUMBER, OP_RJUMP, 0xff, 0xfb, OP_POP, OP_STOP))) == 1
+    assert validate_function(0, bytes((OP_NUMBER, OP_POP, OP_NUMBER, OP_RJUMP, 0xff, 0xfc, OP_POP, OP_STOP))) == -102
+
+def test_conditional_jump():
+    # Each branch ending with STOP
+    assert validate_function(0, bytes((OP_PUSH1, 0xff, OP_PUSH1, 0x01, OP_RJUMPI, 0x00, 0x02, OP_POP, OP_STOP, OP_POP, OP_STOP))) == 2
+    # One branch ending with RJUMP
+    assert validate_function(0, bytes((OP_PUSH1, 0xff, OP_PUSH1, 0x01, OP_RJUMPI, 0x00, 0x04, OP_POP, OP_RJUMP, 0x00, 0x01, OP_POP, OP_STOP))) == 2
+    # Fallthrough
+    assert validate_function(0, bytes((OP_PUSH1, 0xff, OP_PUSH1, 0x01, OP_RJUMPI, 0x00, 0x04, OP_DUP1, OP_DUP1, OP_POP, OP_POP, OP_POP, OP_STOP))) == 3
+    # Offset 0
+    assert validate_function(0, bytes((OP_PUSH1, 0xff, OP_PUSH1, 0x01, OP_RJUMPI, 0x00, 0x00, OP_POP, OP_STOP))) == 2    
+    # Simple loop (RJUMP offset = -5)
+    assert validate_function(0, bytes((OP_PUSH1, 0x01, OP_PUSH1, 0xff, OP_DUP2, OP_SUB, OP_DUP1, OP_RJUMPI, 0xff, 0xfa, OP_POP, OP_POP, OP_STOP))) == 3
+    # One branch increasing max stack more stack than another
+    assert validate_function(0, bytes((OP_PUSH1, 0x01, OP_RJUMPI, 0x00, 0x07, OP_ADDRESS, OP_ADDRESS, OP_ADDRESS, OP_POP, OP_POP, OP_POP, OP_STOP, OP_ADDRESS, OP_POP, OP_STOP))) == 3
+    assert validate_function(0, bytes((OP_PUSH1, 0x01, OP_RJUMPI, 0x00, 0x03, OP_ADDRESS, OP_POP, OP_STOP, OP_ADDRESS, OP_ADDRESS, OP_ADDRESS, OP_POP, OP_POP, OP_POP, OP_STOP))) == 3
+
+    # Missing stack argument
+    assert validate_function(0, bytes((OP_RJUMPI, 0x00, 0x00, OP_STOP))) == -101
+    # Stack underflow in one branch
+    assert validate_function(0, bytes((OP_PUSH1, 0xff, OP_PUSH1, 0x01, OP_RJUMPI, 0x00, 0x02, OP_POP, OP_STOP, OP_SUB, OP_POP, OP_STOP))) == -101
+    assert validate_function(0, bytes((OP_PUSH1, 0xff, OP_PUSH1, 0x01, OP_RJUMPI, 0x00, 0x02, OP_SUB, OP_STOP, OP_NOT, OP_POP, OP_STOP))) == -101
+    # Stack not empty in the end of one branch
+    assert validate_function(0, bytes((OP_PUSH1, 0xff, OP_PUSH1, 0x01, OP_RJUMPI, 0x00, 0x02, OP_POP, OP_STOP, OP_NOT, OP_STOP))) == -103
+    assert validate_function(0, bytes((OP_PUSH1, 0xff, OP_PUSH1, 0x01, OP_RJUMPI, 0x00, 0x02, OP_NOT, OP_STOP, OP_POP, OP_STOP))) == -103
+
+def test_unreachable():
+    # Max stack not changed by unreachable code
+    assert validate_function(0, bytes((OP_ADDRESS, OP_POP, OP_STOP, OP_ADDRESS, OP_ADDRESS, OP_ADDRESS, OP_POP, OP_POP, OP_POP, OP_STOP))) == 1
+    assert validate_function(0, bytes((OP_ADDRESS, OP_POP, OP_RJUMP, 0x00, 0x06, OP_ADDRESS, OP_ADDRESS, OP_ADDRESS, OP_POP, OP_POP, OP_POP, OP_STOP))) == 1
+    # Stack underflow in unreachable code
+    assert validate_function(0, bytes((OP_ADDRESS, OP_POP, OP_STOP, OP_POP, OP_STOP))) == 1
+    assert validate_function(0, bytes((OP_ADDRESS, OP_POP, OP_RJUMP, 0x00, 0x01, OP_POP, OP_STOP))) == 1
+
+
+
+test_empty()
```

Comment:
This is not needed if you use `pytest`. It will find all files with `_test` suffix and there all functions with `test_` prefix. I suggest we start using `pytest` officially.